common/gtk-3.0/3.20/render-assets.sh

```diff
@@ -1,34 +1,12 @@
-#! /bin/bash
+#!/bin/bash
+set -ueo pipefail
 
-INKSCAPE="/usr/bin/inkscape"
-OPTIPNG="/usr/bin/optipng"
-
-SRC_FILE="assets.svg"
-ASSETS_DIR="assets"
-INDEX="assets.txt"
-
-for i in `cat $INDEX`
-do 
-if [ -f $ASSETS_DIR/$i.png ]; then
-    echo $ASSETS_DIR/$i.png exists.
+# Make sure that parallel is GNU parallel and not moreutils.
+# Otherwise, it fails silently. There's no smooth way to detect this.
+if [[ "$(which parallel 2> /dev/null)" ]]; then
+  cmd=(parallel)
 else
-    echo
-    echo Rendering $ASSETS_DIR/$i.png
-    $INKSCAPE --export-id=$i \
-              --export-id-only \
-              --export-png=$ASSETS_DIR/$i.png $SRC_FILE >/dev/null \
-    && $OPTIPNG -o7 --quiet $ASSETS_DIR/$i.png 
+  cmd=(xargs -n1)
```

Comment:
Can't we decide on one of the two methods? Is it really required to have two parallel tools to support? If someone has not got the one I use and we run into problems its harder to debug.

Reply:
if you like spending `nproc` times more time on generating the theme there is no problem to use do in parallel or not :D

Reply:
I think I missunderstood the command. Nevermind my comment.